packages/trader/src/App/Components/Layout/Header/platform-switcher.jsx

```diff
@@ -0,0 +1,52 @@
+import PropTypes            from 'prop-types';
+import React                from 'react';
+import { CSSTransition }    from 'react-transition-group';
+import { localize }         from 'App/i18n';
+import Icon                 from 'Assets/icon.jsx';
+import { PlatformDropdown } from './platform-dropdown.jsx';
+import 'Sass/app/_common/components/platform-switcher.scss';
+
+class PlatformSwitcher extends React.PureComponent {
+    constructor(props) {
+        super(props);
+        this.state = { is_open: false };
+    }
+
+    handleClick = () => this.setState({ is_open: !this.state.is_open });
+
+    render = () => (
+        <React.Fragment>
+            <CSSTransition
```

Comment:
@brandon-binary do you need to use CSSTransition for the `platform_switcher` div? if it's only to animate the arrow, i suggest to add a classname to the arrow instead